AcceptanceEKSJenkinsfile

```diff
@@ -58,6 +62,10 @@ pipeline {
                     source uaa/scripts/setup-tests.sh
 
                     export JRE_VERSION="{ jre: { version: 11.+ }}"
+                    export ARTIFACTORY_READER=$FUNCTIONAL_USER_USR
+                    export ARTIFACTORY_READER_PW=$FUNCTIONAL_USER_PSW
+                    export SAML_IDP_USER=$FUNCTIONAL_USER_USR
+                    export SAML_IDP_USER_PW=$FUNCTIONAL_USER_PSW
```

Comment:
So the SAML IDP credentials are the same as those for Artifactory reader?

Reply:
Yes, we are using the same GE SSO functional user